### ISSUE TITLE
Staging Review finding: Link in disclaimer to user guide does not use descriptive link text

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#105793]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** EDM and VEBT team
**Product:** GI Bill Comparison Tool
**Feature:** GI Bill Comparison Tool homepage

## Findings details
**VA.gov Experience Standard - issue:** [User knows where a button or link leads.](https://design.va.gov/about/experience-standards/#user-knows-where-a-button-or-link-leads)
**VA.gov Experience Standard - category:** [Comprehension](https://design.va.gov/about/experience-standards/#category-comprehension)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @erinrwhite (IA), @briandeconinck (Accessibility), @kkitagawa-bosch (content)

### Description

The link text for the new user guide in the disclaimer has the link text `(opens in a new tab)` and nothing else. The link text should be clear and indicate where the link goes. 

<details><summary>Link, screenshot or steps to recreate</summary>

<img width="496" alt="Image" src="https://github.com/user-attachments/assets/240ccecc-2117-409e-b4fe-63799de9a616" />

</details>

## Recommended action

Update the link text to clearly indicate where the link will go, and indicate that it's opening a PDF file.

## References

- **Accessibility Defect Severity:** [2: High. Should be fixed in 1-2 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-2)
- **WCAG Success Criteria:** [WCAG 2.4.9 Link Purpose (Link Only) - Level AAA](https://www.w3.org/TR/WCAG21/#link-purpose-link-only)
- **Modality:** Screen reader
- **Design System Component:** [Link component](https://design.va.gov/components/link)
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** [Links](https://design.va.gov/content-style-guide/links)
- **Context:** Navigation

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by jkdrahota** [comment]
This update has been made.
